test/integration/webhook/jobs/mpijob_webhook_test.go

```diff
@@ -23,24 +23,21 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"sigs.k8s.io/kueue/pkg/controller/jobs/mpijob"
 	testingjob "sigs.k8s.io/kueue/pkg/util/testingjobs/mpijob"
-	"sigs.k8s.io/kueue/test/integration/framework"
 	"sigs.k8s.io/kueue/test/util"
 )
 
-var _ = ginkgo.Describe("MPIJob Webhook", func() {
+var _ = ginkgo.Describe("MPIJob Webhook", ginkgo.Ordered, func() {
```

Comment:
Whys it the structure for the Describe and When changed (the Ordered label moved from When to Describe)? Also, I don't see ContinueOnFailure anymore. Is it needed in this PR?

Reply:
In our test `ginkgo.Ordered` is label to decorate a container in which we need to be able to call `ginkgo.BeforeAll` (especially in this Spec which has a single `It`, it has no other value),  The `BeforeAll` was moved just for to me this look similar to the other webhook specs.

Reply:
sgtm, though I prefer to see drive-by cleanups in a separate PR to reduce distractions. Anyway, I can accept it after I now understand.